### PR TITLE
Fix update

### PR DESCRIPTION
**Pull request by aerijman**
work with newer versions of pysam, print header and skip unmapped.

### CODE REVIEW

File: mark-nonconverted-reads.py [hunk 1]
```diff
@@ -92,23 +92,29 @@ def parse_bam(bam_file, fasta_dict, out, args):
         nonconverted_dict[chrom] = 0
         
     for read in bam_file:
-    
-        # Ignore reads not part of a proper pair
+
+        if read.reference_id == -1:
```
Comment (bwlang) [inline mark-nonconverted-reads.py:96]: why is this needed?

**Review by bwlang — APPROVED** [review]

File: mark-nonconverted-reads.py (hunk 1 above)
Comment (aerijman) [reply mark-nonconverted-reads.py:96]: I struggled with a bam with unmapped reads and pysam's error messages were not very helpful... I can take it out if we should NEVER encounter these reads?
